doc/source/api.rst

```diff
@@ -29,7 +29,7 @@ Bolts
 -----
 
 .. autoclass:: streamparse.bolt.Bolt
-    :members: ack, emit, emit_many, fail, initialize, process, log, auto_anchor, auto_ack, auto_fail
+    :members: ack, emit, emit_many, fail, initialize, process, process_tick, log, auto_anchor, auto_ack, auto_fail
```

Comment:
Not that this is @cabiad's fault, but why aren't we just using `:members:` by itself here, which will automatically update?